### ISSUE TITLE
"Unknown keys passed to possession.setup" appears on startup

### ISSUE THREAD

**Issue by ovk** [jedrzejboczar/possession.nvim#27]
After recent update the following error appears on startup:

```
Unknown keys passed to possession.setup:
  plugins.close_windows.match.filetype.1
```

This is my setup:

```lua
possession.setup({
    silent = true,
    autosave = {
        current = true
    },
    plugins = {
        delete_hidden_buffers = {
            hooks = {
            }
        },
        delete_buffers = true,
        close_windows = {
            hooks = { 'before_save', 'before_load' },
            preserve_layout = false,
            match = {
                floating = true,
                buftype = {},
                filetype = { 'aerial' }
            }
        },
    }
})
```

It seems like this was introduced by commit c65d25a . Not sure why this is considered "unknown option" as this is documented option which seems to continue to work.

**Comment by jedrzejboczar** [comment]
Thanks for report, there was a bug in the implementation and it tried to warn about unknown list-like table keys (`1` in this example). It should be fixed now with https://github.com/jedrzejboczar/possession.nvim/commit/fda62baba9fc547658b3546f919127f1363e0cb3.

**Comment by ovk** [comment]
Thank you, @jedrzejboczar . The issue is indeed fixed in fda62ba.
